src/App.php

```diff
@@ -198,6 +199,16 @@ function () use ( $slim ) {
 					$page( 'callback' );
 				} )->name( 'oauth_callback' );
 			} );
+		$slim->get( '/index.css',
+			function () use ( $slim ) {
+				// Compile LESS if need be, otherwise serve cached asset
+				// Cached files get automatically deleted if they are over a week old
+				$lessFiles = array( APP_ROOT . '/src/Less/index.less' => '/copypatrol/' );
+				$options = array( 'cache_dir' => APP_ROOT . '/src/Less/cache' );
+				$cssFileName = Less_Cache::Get( $lessFiles, $options );
+				$slim->response->headers->set( 'Content-Type', 'text/css' );
+				$slim->response->setBody( file_get_contents( APP_ROOT . '/src/Less/cache/' . $cssFileName ) );
```

Comment:
This is where the magic happens. Requests to `index.css` will have to go through the stack, but the file is cached until the `.less` file is changed, so this should have noticeable affect on page load time.

We only to compile `index.less`. From there we can import Bootstrap or any other Less files, but saving that for another PR :)

I could not get a `/css/index.css` route to work, for whatever reason, figured `index.css` works just as well, especially since once we start using bootstrap.less we'd only have one .css file anyway.